tests/func/test_api.py

```diff
@@ -72,6 +122,7 @@ def test_open(remote_url, tmp_dir, dvc):
 @pytest.mark.parametrize("remote_url", all_remote_params, indirect=True)
 def test_open_external(remote_url, erepo_dir, setup_remote):
     setup_remote(erepo_dir.dvc, url=remote_url)
+    ensure_dir_scm(erepo_dir.dvc, remote_url)
```

Comment:
@efiop I've added `ensure_dir_scm` here based off `setup_remote`.

<details>

- Using `ensure_dir` instead of `ensure_dir_scm`, there's
`ERROR: configuration error - config file error: remote 'upstream' doesn't exists.`
- Using `run_dvc("remote", "add", ...)` instead results in
`ERROR: configuration error - config file error: remote 'upstream' already exists. Use `-f|--force` to overwrite it.`

</details>